charts/testkube-enterprise/templates/dex-config-secret.yaml

```diff
@@ -29,11 +29,13 @@ stringData:
         secret: '{{ $api.api.oauth.clientSecret }}'
         redirectURIs:
           - '{{ if .Values.global.domain }}https://{{ .Values.global.restApiSubdomain }}.{{ .Values.global.domain }}/auth/callback{{ else }}http://localhost:8090/auth/callback{{ end }}'
+          - '{{ if .Values.global.domain }}https://{{ .Values.global.restApiSubdomain }}.{{ .Values.global.domain }}/auth/callback{{ else }}http://localhost:38090/auth/callback{{ end }}'
```

Comment:
Maybe this a bit better:

 redirectURIs:
 {{- if .Values.global.domain }}
     - https://{{ .Values.global.restApiSubdomain }}.{{ .Values.global.domain }}/auth/callback
   {{- else }}
     - http://localhost:8090/auth/callback
     - http://localhost:38090/auth/callback
  {{- end }}